pong.cpp

```diff
@@ -20,15 +22,110 @@ void Pong::PongInit()
     auto ball = std::make_unique<Ball>();
     ball->OnStart();
     mGameObjects.push_back(std::move(ball));
+
+    const float horizontalWallWidth = 640 * 2;
+    const float horizontalWallHeight = 25;
+    const float horizontalWallY = 480;
+    const float verticalWallWidth = 25;
+    const float verticalWallHeight = 480 * 2;
+    const float verticalWallX = 640;
```

Comment:
all of the above (especially the multiplications), could be constexpr and calculated at compile time